lib/webdriver-user-agent/browser_options.rb

```diff
@@ -32,25 +36,68 @@ def options
         @options ||= {}
       end
 
+      def set_preview_option(opt)
+        @stp = opt
+        @stp = false unless @stp.is_a?(TrueClass)
```

Comment:
We can replace line 40 & 41 with `@stp = !!(opt)`

Reply:
I like your drive to reducing complexity. However, I'd like to be pessimistic about the developer intent for engaging this non-mainstream option (which, as of my testing, does not actually work), and check that the option is exactly a TrueClass object (and not `"false"` or `123.45`, which evaluate to `true` if parsed in `!!(opt)`). What do you think, is my pessimism necessary or no?

Reply:
Yup, you're right. Checking whether the object is TrueClass is a better option.